recipes/python-magic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "python-magic" %}
+{%set version = "0.4.12" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "a04b20900100884d4fce40a767182a16fcb9d10756c67cdc21f5fa610b7c9d3c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    # - magic
+    # This script doesn't install libmagic itself,
+    # so the python bindings will fail if imported
```

Comment:
That's why it's WIP. Need to add those scripts...

On Fri, Aug 26, 2016, 12:27 AM jakirkham <notifications@github.com> wrote:

> In recipes/python-magic/meta.yaml
> <https://github.com/conda-forge/staged-recipes/pull/1392#discussion_r76375039>
> :
>
> > +  number: 0
> > +  script: python setup.py install --single-version-externally-managed --record=record.txt
> > +
> > +requirements:
> > +  build:
> > +    - python
> > +    - setuptools
> > +
> > +  run:
> > +    - python
> > +
> > +test:
> > +  imports:
> > +    # - magic
> > +    # This script doesn't install libmagic itself,
> > +    # so the python bindings will fail if imported
>
> So how do we test this then?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/1392/files/73ca914a97da55beadee72201029f729aed106df#r76375039>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEGphJmcDoJC0DmIcWvDvK0bEtvOOMkRks5qjpVPgaJpZM4JtzFm>
> .
>